src/crypto/README.md

```diff
@@ -0,0 +1,68 @@
+interface-crypto
+==================
+
+> A test suite you can use to implement a libp2p crypto module. A libp2p crypto module is used to ensure all exchanged data between two peers is encrypted.
+
+**Modules that implement the interface**
```

Comment:
Is plaintext not going to be a module?

Reply:
Yes, that's the intent, I'm planning on writing that and the private network refactor this week. I was planning on just bundling plaintext 2 with libp2p though, and linking to it here.

Reply:
Ok, perfect. I'll keep this discussion open until that's resolved for tracking.

Reply:
I also created https://github.com/libp2p/js-interfaces/issues/3 to track adding plaintext.